### ISSUE TITLE
Question (not a bug)

### ISSUE THREAD

**Issue by g3nsvrv** [rodepanda/SpiceCompanion#3]
hi guys, I just want to congrats you for this project, I was testing a lot of things for my phone, even something that is available on the store right now which is not working for me for some reason (i tried the api itself with a web server that it is working), and yeah, thanks guys.

I wanna ask you something, do you have anything like a resource where I can get the methods and things that the API needs to run?

Basically I was trying to use curl to do things but I get HTTP not allowed, watching your code I suspect that only accepts JSON as a POST.

**Comment by rodepanda** [comment]
The api doesn't use http. You need to open a socket and send json in utf-8 format terminated with a null byte.
The documentation can be found in the Spice source code Readme.md.
If the app is not working on your phone, try restarting it.

**Comment by g3nsvrv** [comment]
Regarding to the last one, I will open a new thread for that. thanks @rodepanda for the clarifications btw.
